lego-webapp/components/DisplayContent/index.tsx

```diff
@@ -5,7 +5,7 @@ import styles from './DisplayContent.module.css';
 import type { CSSProperties } from 'react';
 
 type Props = {
-  content: string;
+  content: string | undefined;
```

Comment:
I believe this is the same as 
```suggestion
  content?: string;
```